packages/test/src/admin-utils.ts

```diff
@@ -189,3 +196,33 @@ export async function createPromotion(
   });
   return createPromotion as Promotion;
 }
+
+export async function getSuperadminContextInChannel(
```

Comment:
I see you are using this helper in the source code of the assign products script. In that case, can you move it to `packages/util` instead of `packages/test`? `Packages/test` should only contain helpers that are used in tests

Reply:
It is actually used by both tests and src code. Where should it belong in that case? @martijnvdbrug

Reply:
In that case I would say `packages/util`. 

The main reason that the folder ends up in the compiled `dist`:
* `dist/util/src/...`
* `dist/vendure-scripts/src/..`

While its already not too great that we have `src` in dist, it will confuse other devs even more if we have a folder named `test` in the dist :-)